eip-0027.md

```diff
@@ -84,12 +84,6 @@ Contracts
 
     val correctNftId = EQ(firstTokenId, ByteArrayConstant(reemissionNftId))
     
-    // output of the reemission contract
```

Comment:
why removed? reemissionOut and minerOut used below: 

```
    // miner's output must have a script which is the time-locking reward for the miner's pubkey
    // box height must be the same as block height
    val correctMinerOutput = AND(
      EQ(ExtractScriptBytes(minerOut), expectedMinerOutScriptBytesVal(monetarySettings.minerRewardDelay, MinerPubkey)),
      EQ(Height, boxCreationHeight(minerOut))
    )
    
    // reemission output's height must be the same as block height
    val heightCorrect = EQ(boxCreationHeight(reemissionOut), Height)
```

Reply:
But they are declare twice, see above.